### ISSUE TITLE
连接不上web_demo界面

### ISSUE THREAD

**Issue by moliceonly** [FunAudioLLM/Fun-Audio-Chat#16]
TTS应该是正常启动的，然后在跟着客户端的启动指引操作也是正常，但是最后无论是访问https://localhost:80/ 还是https://172.17.0.5:80/都是一直加载很难连接上，唯一一次连接上出现了moshi.chat 页面，但是也无法正常使用，要怎么解决？

(FunAudioChat) root@7391c5ab5455:/media/SSDDisk2/yexuanting/04_Full_Duplex/Fun-Audio-Chat# CUDA_VISIBLE_DEVICE3 python -m web_demo.server.server --model-path pretrained_models/Fun-Audio-Chat-8B --port 11236 --tts-gpu 0
[INFO] Initializing server with Opus sample rate: 24000, Model sample rate: 16000
[INFO] loading s2s model to cuda...
Loading checkpoint shards: 100%|██████████████████████████████████████████████████| 4/4 [00:00<00:00, 52.36it/s]
[INFO] s2s model loaded (: cuda)
[INFO] waiting for tts model loading ... 
[INFO] Output directory: ./output
[INFO] Global TTS process started (pid: 1543086)
[INFO] Access the Web UI directly at http://localhost:11236
2025-12-28 10:03:16,061 DEBUG Using selector: EpollSelector
======== Running on http://localhost:11236 ========
(Press CTRL+C to quit)
[INFO] [TTS Process] Starting TTS worker process on cuda:0...
[INFO] [TTS Process] Loading TTS model...
2025-12-28 10:03:21.921 | INFO     | utils.cosyvoice_detokenizer:get_audio_detokenizer:87 - cosyvoice cuda: True
/opt/conda/envs/FunAudioChat/lib/python3.12/site-packages/lightning/fabric/__init__.py:41: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html. The pkg_resources package is slated for removal as early as 2025-11-30. Refrain from using this package or pin to Setuptools<81.
2025-12-28 10:03:24,804 INFO input frame rate=25
/opt/conda/envs/FunAudioChat/lib/python3.12/site-packages/torch/nn/utils/weight_norm.py:144: FutureWarning: `torch.nn.utils.weight_norm` is deprecated in favor of `torch.nn.utils.parametrizations.weight_norm`.
  WeightNorm.apply(module, name, dim)
2025-12-28 10:03:27.637984996 [W:onnxruntime:, transformer_memcpy.cc:74 ApplyImpl] 14 Memcpy nodes are added to the graph main_graph for CUDAExecutionProvider. It might have negative impact on performance (including unable to run CUDA graph). Set session_options.log_severity_level=1 to see the detail logs before this message.
2025-12-28 10:03:27.643861557 [W:onnxruntime:, session_state.cc:1166 VerifyEachNodeIsAssignedToAnEp] Some nodes were not assigned to the preferred execution providers which may or may not have an negative impact on performance. e.g. ORT explicitly assigns shape related ops to CPU to improve perf.
2025-12-28 10:03:27.643878551 [W:onnxruntime:, session_state.cc:1168 VerifyEachNodeIsAssignedToAnEp] Rerunning with verbose output on a non-minimal build will show node assignments.
2025-12-28 10:03:28,023 DEBUG Attempting to acquire lock 128420157233232 on /root/.cache/modelscope/hub/.lock/pengzhendong___wetext
2025-12-28 10:03:28,024 DEBUG Lock 128420157233232 acquired on /root/.cache/modelscope/hub/.lock/pengzhendong___wetext
2025-12-28 10:03:28,027 DEBUG Starting new HTTPS connection (1): www.modelscope.cn:443
2025-12-28 10:03:28,485 DEBUG https://www.modelscope.cn:443 "GET /api/v1/repos/internalAccelerationInfo HTTP/1.1" 200 188
2025-12-28 10:03:28,487 DEBUG Starting new HTTP connection (1): 100.100.100.200:80
2025-12-28 10:03:28,490 DEBUG Starting new HTTPS connection (1): www.modelscope.cn:443
2025-12-28 10:03:29,907 DEBUG https://www.modelscope.cn:443 "GET /api/v1/models/pengzhendong/wetext HTTP/1.1" 200 None
Downloading Model from https://www.modelscope.cn to directory: /root/.cache/modelscope/hub/models/pengzhendong/wetext
2025-12-28 10:03:30,796 DEBUG https://www.modelscope.cn:443 "GET /api/v1/models/pengzhendong/wetext/revisions HTTP/1.1" 200 222
2025-12-28 10:03:31,133 DEBUG https://www.modelscope.cn:443 "GET /api/v1/models/pengzhendong/wetext/repo/files?Revision=master&Recursive=True HTTP/1.1" 200 None
2025-12-28 10:03:31,134 DEBUG Attempting to release lock 128420157233232 on /root/.cache/modelscope/hub/.lock/pengzhendong___wetext
2025-12-28 10:03:31,134 DEBUG Lock 128420157233232 released on /root/.cache/modelscope/hub/.lock/pengzhendong___wetext
2025-12-28 10:03:31,243 DEBUG Attempting to acquire lock 128410910863392 on /root/.cache/modelscope/hub/.lock/pengzhendong___wetext
2025-12-28 10:03:31,243 DEBUG Lock 128410910863392 acquired on /root/.cache/modelscope/hub/.lock/pengzhendong___wetext
2025-12-28 10:03:31,244 DEBUG Starting new HTTPS connection (1): www.modelscope.cn:443
2025-12-28 10:03:31,493 DEBUG https://www.modelscope.cn:443 "GET /api/v1/repos/internalAccelerationInfo HTTP/1.1" 200 188
2025-12-28 10:03:31,495 DEBUG Starting new HTTP connection (1): 100.100.100.200:80
2025-12-28 10:03:31,497 DEBUG Starting new HTTPS connection (1): www.modelscope.cn:443
2025-12-28 10:03:32,665 DEBUG https://www.modelscope.cn:443 "GET /api/v1/models/pengzhendong/wetext HTTP/1.1" 200 None
Downloading Model from https://www.modelscope.cn to directory: /root/.cache/modelscope/hub/models/pengzhendong/wetext
2025-12-28 10:03:33,481 DEBUG https://www.modelscope.cn:443 "GET /api/v1/models/pengzhendong/wetext/revisions HTTP/1.1" 200 222
2025-12-28 10:03:33,771 DEBUG https://www.modelscope.cn:443 "GET /api/v1/models/pengzhendong/wetext/repo/files?Revision=master&Recursive=True HTTP/1.1" 200 None
2025-12-28 10:03:33,773 DEBUG Attempting to release lock 128410910863392 on /root/.cache/modelscope/hub/.lock/pengzhendong___wetext
2025-12-28 10:03:33,773 DEBUG Lock 128410910863392 released on /root/.cache/modelscope/hub/.lock/pengzhendong___wetext
2025-12-28 10:03:33,876 INFO use wetext frontend
2025-12-28 10:03:37.791 | INFO     | utils.cosyvoice_detokenizer:get_audio_detokenizer:91 - cosyvoice loaded
[INFO] [TTS Process] TTS model loaded successfully

(FunAudioChat) root@7391c5ab5455:/media/SSDDisk2/yexuanting/04_Full_Duplex/Fun-Audio-Chat/web_d
emo/client# nvm use
Found '/media/SSDDisk2/yexuanting/04_Full_Duplex/Fun-Audio-Chat/web_demo/client/.nvmrc' with version <v22.20.0>
Now using node v22.20.0 (npm v10.9.3)
(FunAudioChat) root@7391c5ab5455:/media/SSDDisk2/yexuanting/04_Full_Duplex/Fun-Audio-Chat/web_d
emo/client# openssl req -x509 -newkey rsa:4096 -keyout key.pem -out cert.pem -days 365 -nodes
.+.....+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*..........+.........+................+.....+.........+....+.....+...+.......+...+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*......+.............................+.........+..................................+...........+.+...........+...+....+...+..+.............+...+.....+.......+.....+.........................+...+..+.............+......+...+........+.......+.....+...............+......+....+..+.........+..........+.....+....+...............+........+..........+..+..........+......+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
....+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*...+.........+..+.......+.....+.+..+....+...+..+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*.+.....+....+.....+.+..+.........+................+...+.........+.....+.......+........+...+....+........+..................+.........+.+..+..........+........+...+.+...+...+...+.....+..............................+.......+.....+.......+......+..............+.......+........+..........+...+..+..........+.....+.+.....+.......+........+.+.....+...................+..+.......+...+.........+.....+....+..+.............+..+............+...............+...+...+.+......+..+...+...............+.......+.....+............+...+......+.+......+........+..........+...........+.+.........+......+..+...+.............+..+.......+...............+..+......+.............+.........+..+.....................+.......+..+..........+.......................+..........+......+.....+....+...........+.+.....+.+...+..............+.+.........+.....+......+......+....+.........+..+.+.....+....+...+.....+..........+.....+.+............+..............+...+.+...........+...............+.+.....+.........+.+..+.......+...+........+...+.+...........+...+.+...+.....+.+......+.....+...+......+...............+.+..................+...............+........+...+....+...............+..+....+.....+......+....+.........+.................+......+.......+.....+............+......+...+.+......+...+..+....+......+.........+......+........+......+......+..................+..........+.................+.......+...........+....+.....+.+.....+.+......+........+.........+..................+............+.......+...+..+.............+........+.........+.........+.+.....+....+..+...............+....+.........+.....+....+......+..+.............+......+.....................+..+.......+........+...+....+......+..............+......+...+..........+...............+.....+.+........+...+....+......+..................+..+.......+............+...+...........+..........+..+....+.........+........................+...+...+...............+............+..+.......+..................+..+......+.............+............+...+...........+.........+............+.......+..+.+.....+...+....+..+.+...+..+.......+.....+.+.............................+...+..........+....................+....+...+...............+..+.+........+......+........................+.+........+.+..+...+.+...............+.....+....+.........+...+..................+...........+......+....+.........+..+...............+.........+..........+......+.........+...+...+.........+...+...........+.......+...+..+.........+...+...+..........+.................+.+.....+............+.......+.....+.......+...........+...+...................+.....+.......+........+..........+...+....................+.........+......+...+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
-----
You are about to be asked to enter information that will be incorporated
into your certificate request.
What you are about to enter is what is called a Distinguished Name or a DN.
There are quite a few fields but you can leave some blank
For some fields there will be a default value,
If you enter '.', the field will be left blank.
-----
Country Name (2 letter code) [AU]:
State or Province Name (full name) [Some-State]:
Locality Name (eg, city) []:
Organization Name (eg, company) [Internet Widgits Pty Ltd]:
Organizational Unit Name (eg, section) []:
Common Name (e.g. server FQDN or YOUR name) []:
Email Address []:
(FunAudioChat) root@7391c5ab5455:/media/SSDDisk2/yexuanting/04_Full_Duplex/Fun-Audio-Chat/web_d
emo/client# npm install

added 1 package, and audited 407 packages in 651ms

158 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
(FunAudioChat) root@7391c5ab5455:/media/SSDDisk2/yexuanting/04_Full_Duplex/Fun-Audio-Chat/web_d
emo/client# npm run dev

> kyutai-client@0.0.0 dev
> vite

11:06:24 AM [vite] (client) Re-optimizing dependencies because lockfile has changed

  VITE v7.3.0  ready in 247 ms

  ➜  Local:   https://localhost:80/
  ➜  Network: https://172.17.0.5:80/
  ➜  press h + enter to show help

<img width="482" height="68" alt="Image" src="https://github.com/user-attachments/assets/3ef3fa92-c713-44c6-b2db-73816f551e6a" />

**Comment by Junehom** [comment]
后端服务不起起来应该前端也是能正常显示的，只会在半双工连接跳转之后没有正常对话。server log看起来正常，暂时应该是纯前端的问题，连接不上看看1. 有没有挂代理什么的，2. 尝试vite.config.js里面port: 80换成一个其他的端口比如5374之类不会有冲突的，3. 如果是docker环境的话看看有没有将端口映射出去
